src/features/logical-assignment.md

```diff
@@ -0,0 +1,90 @@
+Logical assignment feature explainer
+
+---
+title: 'Logical assignment'
+author: 'Shu-yu Guo ([@_shu](https://twitter.com/_shu))'
+avatars:
+  - 'shu-yu-guo'
+date: 2020-04-22
+tags:
+  - ECMAScript
+description: 'JavaScript now supports compound assignment with logical operations.'
+tweet: 'TODO'
```

Comment:
Please leave as empty string, otherwise this will generate invalid URL.

Reply:
Done.